.travis.yml

```diff
@@ -1,7 +1,9 @@
 language: php
 php:
-  - 7.2
   - 5.6
```

Comment:
5.6 is no longer supported.
```suggestion
```

Reply:
24 more days...
http://php.net/supported-versions.php